src/interface/meta.thrift

```diff
@@ -141,13 +141,19 @@ struct EdgeItem {
     4: Schema           schema,
 }
 
+struct IndexParams {
```

Comment:
Rename to S2IndexParams?

Reply:
Using `IndexParams` is ok because we may add other index params for other datatypes in the future.

Reply:
ACK.